package-lock.json

```diff
@@ -266,9 +266,9 @@
       "integrity": "sha512-Gd2UZBJDkXlY7GbJxfsE8/nvKkUEU1G38c1siN6QP6a9PT9MmHB8GnpscSmMJSoF8LOIrt8ud/wPtojys4G6+g=="
     },
     "source-map": {
-      "version": "0.5.7",
-      "resolved": "https://registry.npmjs.org/source-map/-/source-map-0.5.7.tgz",
-      "integrity": "sha1-igOdLRAh0i0eoUyA2OpGi6LvP8w="
+      "version": "0.5.6",
```

Comment:
why degrade the version?

Reply:
because the version we have checked in and were using was 0.5.6.

We happened to also have 0.5.7 transitively installed in node_modules here too.